launcher/src/CmderLauncher.cpp

```diff
@@ -153,17 +155,26 @@ void StartCmder(std::wstring path, bool is_single_mode)
 		swprintf_s(args, L"/Icon \"%s\" /Title Cmder", icoPath);
 	}
 
-		SetEnvironmentVariable(L"CMDER_ROOT", exeDir);
-		if (!streqi(path.c_str(), L""))
-		{
-			SetEnvironmentVariable(L"CMDER_START", path.c_str());
+	SetEnvironmentVariable(L"CMDER_ROOT", exeDir);
+	if (!streqi(path.c_str(), L""))
+	{
+		if (!SetEnvironmentVariable(L"CMDER_START", path.c_str())) {
+			MessageBox(NULL, _T("Error trying to set CMDER_START to given path!"), _T("Error"), MB_OK);
 		}
-
-		// Send out the Settings Changed message - Once using ANSII...
-		//SendMessageTimeout(HWND_BROADCAST, WM_SETTINGCHANGE, 0, (LPARAM)"Environment", SMTO_ABORTIFHUNG, 5000, NULL);
-
-		// ...and once using UniCode (because Windows 8 likes it that way).
-		//SendMessageTimeout(HWND_BROADCAST, WM_SETTINGCHANGE, 0, (LPARAM) L"Environment", SMTO_ABORTIFHUNG, 5000, NULL);
+	}
+	else
+	{
+		wchar_t* homeProfile = 0;
+		SHGetKnownFolderPath(FOLDERID_Profile, 0, NULL, &homeProfile);
+		if (!SetEnvironmentVariable(L"CMDER_START", homeProfile)) {
```

Comment:
IM this is still a bad change: init.bat will now *always* set the path to Home if cmder isn't started with a path. This means that startup path in the conemu config is completely useless now :-(

This also means that you can't do what's asked for in https://github.com/cmderdev/cmder/issues/203